lib/blueprinter/view_collection.rb

```diff
@@ -35,7 +35,8 @@ def fields_for(view_name)
     end
 
     def transformers(view_name)
-      views[view_name].transformers
+      included_transformers = gather_transformers_from_included_views(view_name)
+      included_transformers.concat(views[:default].transformers).uniq
```

Comment:
A change from https://github.com/procore-oss/blueprinter/pull/363 is that the view transformers are now merged with the default transformers, instead of replacing them:

https://github.com/procore-oss/blueprinter/blob/e3c8ecc1ae461a6f54ac5c67d0dcd53471b5b1c3/lib/blueprinter/view_collection.rb#L39

This is the behavior I expected, since default fields on the blueprint are merged with the views' fields. Though it's debatable as "default" can be ambiguous.